src/components/CreateWallet.jsx

```diff
@@ -96,14 +107,14 @@ const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
   )
 }
 
-const Seedphrase = ({ seedphrase }) => {
+const Seedphrase = ({ seedphrase, isBlurred }) => {
```

Comment:
```suggestion
const Seedphrase = ({ seedphrase, isBlurred = true }) => {
```
Not necessary, but it would be a nice pro private default.

Reply:
Let's do it.